src/lib/import-export/import/importers/importer.ts

```diff
@@ -215,9 +218,18 @@ abstract class BaseBackupImporter extends BaseImporter {
 		const wpContentDestDir = path.join( rootPath, 'wp-content' );
 		for ( const files of Object.values( wpContent ) ) {
 			for ( const file of files ) {
-				const stats = await lstat( file );
-				// Skip if it's a directory
-				if ( stats.isDirectory() ) {
+				try {
+					const stats = await lstat( file );
+					// Skip if it's a directory
+					if ( stats.isDirectory() ) {
+						continue;
+					}
+				} catch {
+					/**
+					 * If the file does not exist, skip it.
+					 * This can happen if a empty directory is included in the backup
+					 * because the empty directory won't be included in the extraction.
+					 */
```

Comment:
I discovered this issue in a Local export.
The `uploads/2025/01` folder was empty on the site and the `01` folder is missing from the extracted ZIP.
This caused `lstat` to fail because the directory didn't exist.